src/wasm-pack/rust-code.md

```diff
@@ -27,27 +27,13 @@ Now we need to add this to the top of the file:
 
 ```rust
 #![feature(proc_macro, wasm_import_module, wasm_custom_section)]
+
 extern crate wasm_bindgen;
+
 use wasm_bindgen::prelude::*;
 ```
 
-Let's step through this line by line. First up is the list of nightly features. We're enabling this for
-the whole crate. What this means is that we will later tag code with an attribute and this will
-allow Rust to generate code that we don't have to write by hand. In our case it'll use
-`wasm-bindgen.` It should be noted that `#![feature(...)]` implies using the nightly
-compiler. This gated feature will hopefully be stabilized and landed soon so that you won't need it!
-
-`wasm-bindgen` knows how to make code that works well with wasm so we don't have to
-worry about it too much and just write Rust code for the most part. If you want to know the full
-extent of its capabilities check out the README on its repo which can be found
-[here](https://github.com/alexcrichton/wasm-bindgen). For our purposes we need to know that if we
-want functions to work with wasm easily we'll need it.
-
-The next line says we're importing the `wasm-bindgen` crate and the line after that imports the
-prelude from `wasm-bindgen`. The `extern crate` call lets the compiler know what crates to link in
-and the `prelude` contains all the types and functions that `wasm-bindgen` needs to work properly!
-
-Cool let's import the `alert` function from JS so that we can call it in our Rust code!
```

Comment:
I'm unsure whether we want to remove all this and what level of `wasm-bindgen` familiarity the `wasm-pack` tutorial is expecting.

+cc @ashleygwilliams @mgattozzi